src/insns/load_store_c0.adoc

```diff
@@ -0,0 +1,2 @@
+Any instance of this instruction with a `cs1` of `c0` will certainly trap (with a CHERI tag violation), as `c0` is defined to always hold a <<null-cap>>.
+As such, these forms should be considered available for use by future extensions.
```

Comment:
I'd go a little bit further with this, and say RESERVED but allow implementations to take the tag violation, so that they don't actually need to do anything about this until defining new behaviour.

Reply:
A load or store with a `c0` base is something that can happen only if the compiler can statically determine that the address is null, which is undefined behaviour (the compiler will typically emit a trap here instead).  As such, it's not something you'll see accidentally.

If this raises a tag violation, then a trap handler that wants to emulate some future extension that uses this bit of the encoding space needs to pattern match on the opcode and the operand to determine whether this is a reserved instruction or data-dependent error.  That's annoying to have to do because it's extra instructions on the tag-violation path (which is otherwise something that you'd handle much higher up the stack, for example in a signal handler).

Trap and emulate would be better served by these generating a reserved instruction exception, which is where normal trap-and-emulate code paths live.